gslabModel/tests/testthat.R

```diff
@@ -0,0 +1,4 @@
+library(testthat)
```

Comment:
Thanks! I add the instruction on how to just run unit tests in the [README.md](https://github.com/gslab-econ/gslab_r/blob/333d54cd50fb1e8258fe06e612c22aa088aff92a/README.md#tools) and add a [log file](https://github.com/gslab-econ/gslab_r/blob/333d54cd50fb1e8258fe06e612c22aa088aff92a/GSLabModel/tests/tests.log) for test.